picard/formats/asf.py

```diff
@@ -138,6 +138,7 @@ class ASFFile(File):
         'djmixer': 'WM/DJMixer',
         'mixer': 'WM/Mixer',
         'artists': 'WM/ARTISTS',
```

Comment:
jaudiotagger tag mappings doc is out of date actually best to use the spreadsheet link below, the jaudiotagger source code on the songkong help http://www.jthink.net/songkong/help/help.html#d5e2051for uptodate mappings